source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -76,6 +116,25 @@ SplitRequestPtr SimpleRequest::create(ConnPool::Instance& conn_pool,
   return std::move(request_ptr);
 }
 
+SplitRequestPtr SimpleRequest::create(ConnPool::Instance& conn_pool,
```

Comment:
why do we have this separate create method?  Same question for the new create method for other *Request::create(...) too.

Reply:
I'm deleting the duplicates, and it'll be cleaner in the next push...